doc/nvim-tree-lua.txt

```diff
@@ -475,6 +476,10 @@ UI rendering setup
     Compact folders that only contain a single folder into one node in the file tree.
       Type: `boolean`, Default: `false`
 
+    *nvim-tree.renderer.full_name*
+    Display node whose name length is wider than the width of nvim-tree window in floating window.
```

Comment:
Need to specify that this only applies to left trees. Maybe add:

    Only works when |nvim-tree.view.side| is `left`.